x/escrow/time.go

```diff
@@ -12,10 +10,10 @@ import (
 // This function panic if the block time is not provided in the context. This
 // must never happen. The panic is here to prevent from broken setup to be
 // processing data incorrectly.
-func isExpired(ctx weave.Context, t time.Time) bool {
+func isExpired(ctx weave.Context, t weave.UnixTime) bool {
 	blockNow, ok := weave.BlockTime(ctx)
 	if !ok {
 		panic("block time is not present")
 	}
-	return t.Before(blockNow)
+	return t.Time().Before(blockNow)
```

Comment:
This comparison should happen on `weave.UnixTime`, not on `Time`.

E.g. `100.1 before 100.2` is true on Time but `UnixTime(100.1) < UnixTime(100.2)` is false